cvat/settings/base.py

```diff
@@ -667,3 +674,7 @@ class CVAT_QUEUES(Enum):
 }
 
 BUCKET_CONTENT_MAX_PAGE_SIZE =  500
+
+IMPORT_CACHE_FAILED_TTL = timedelta(days=90)
+IMPORT_CACHE_SUCCESS_TTL = timedelta(hours=1)
+RUN_CLEAN_IMPORT_CACHE_FUNC_AFTER = timedelta(hours=2)
```

Comment:
The chose name would better suit for a boolean value, I suggest to rename into something like `UPLOADED_FILES_TTL`, `IMPORT_DATA_TTL`, `IMPORT_CACHE_CLEAN_DELAY` etc., supposing the value is about time.